biosys/apps/main/tests/api/test_dataset.py

```diff
@@ -597,3 +601,99 @@ def test_bulk_delete_all(self):
         resp = self.client.get(self.url, data=None, format='json')
         self.assertEqual(resp.status_code, status.HTTP_200_OK)
         self.assertEqual(len(resp.json()), 0)
+
+class TestDatasetRecordsSearchAndOrdering(TestCase):
+    fixtures = [
+        'test-users',
+        'test-projects',
+        'test-sites',
+        'test-datasets',
```

Comment:
I'm trying to get rid of the datasets fixtures with the uses of some helpers/factories.
Most of them are implemented. I just need to go back to old tests and refactor them.
For examples of the usage of these helpers see `tests.api.test_records_upload.py`
I will do the refactor. Don't worry.

Reply:
Ok, will leave in for now, although kinda useful to have to I can predict what to test against.